lib/mplcursors/_mplcursors.py

```diff
@@ -138,6 +138,7 @@ def __init__(self,
                  multiple=False,
                  highlight=False,
                  hover=False,
+                 interpolated_picking=True,
```

Comment:
I would suggest to make it by default disabled. This is really weird and most of the time your are interested to pick your actual data, but not some fake data...

Reply:
ok